### PR TITLE
011 Multiplication Table up to 12

### PR DESCRIPTION
**Pull request by sajir-dev**

### CODE REVIEW

File: 011_MultiplicationTable.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+multiplicationtable = (n, index) => {
```
Comment (kuism) [inline 011_MultiplicationTable.js]: It's not printed properly. Make sure you are doing everything properly. Don't just do it for the sake.

File: 011_MultiplicationTable.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+multiplicationtable = () => {
+  let table = "";let ch = '  '
```
Comment (kuism) [inline 011_MultiplicationTable.js]: no space after ;

Comment (sajir-dev) [reply 011_MultiplicationTable.js]: Has put the space

Comment (kuism) [reply 011_MultiplicationTable.js]: there is no semi colon after ch = '  '

Comment (sajir-dev) [reply 011_MultiplicationTable.js]: has put semi colon

**Pull request merged**